data.sql

```diff
@@ -1,7 +1,17 @@
+-- Previous INSERT statements
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
-
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8);
-
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
-
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
+
+-- Add new column species
+ALTER TABLE animals ADD COLUMN species CHAR(40);
```

Comment:
👍 Great job adding the `species` column to the `animals` table as per the project requirement! However, it's important to ensure that the query is placed in the correct SQL file for better organization:

- [ ] As this query alters the table structure, it should be included in the `schema.sql` file instead of the `data.sql` file. This separation of schema-altering queries from data-related queries helps maintain a clear structure in your project.

By moving this query to the appropriate file, you'll adhere to best practices and [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Add%20a%20column%20species%20of%20type%20string%20to%20your%20animals%20table.%20Modify%20your%20schema.sql%20file.). Keep up the good work! 👍📜🔧